tests/clients/github/test_githubclient.py

```diff
@@ -24,9 +27,157 @@ def setUp(self):
 
         self.client.username = self.username
         self.client.password = self.password
-
-    def test001_github_client_get_access(self):
-        self.assertEqual(self.client.get_userdata()["emails"][0]["email"], self.email)
+        self.repo_name = ""
 
     def tearDown(self):
         j.clients.github.delete(self.instance_name)
+        if self.repo_name:
+            self.client.delete_repo(repo_name=self.repo_name)
+        if j.sals.fs.exists(path="/tmp/tfttesting/"):
+            j.sals.fs.rmtree("/tmp/tfttesting/")
```

Comment:
Please don't use fixed names.
In case of client used for another user, the files will not be removed.